src/index.js

```diff
@@ -5,4 +5,6 @@ import Main from 'src/pages/Main'
 import 'src/style.css'
 
 
+window.SATURN_VERSION = SATURN_VERSION
```

Comment:
A request: Also write this to the console so I need only open it to see which version I'm using. Feel free to reject.

Reply:
Based on a brief in-person discussion, declining to do this for now in order to reduce console noise. We can always add this later if it would be a big time saver.